From 6aa3a2beedab0ab420d51375860edad3d5d3abbf Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 15 Aug 2019 20:59:29 +0200
Subject: [PATCH 1/1] doc: formatting slimbootloader.rst

Avoid a warning when building the 'make htmldocs' target:

doc/board/intel/slimbootloader.rst:90: WARNING: Title underline too short.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/board/intel/slimbootloader.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/board/intel/slimbootloader.rst b/doc/board/intel/slimbootloader.rst
index 4a46fed069..07c9b126f7 100644
--- a/doc/board/intel/slimbootloader.rst
+++ b/doc/board/intel/slimbootloader.rst
@@ -87,7 +87,7 @@ The PayloadId can be any 4 Bytes value.
    $ qemu-system-x86_64 -machine q35 -nographic -serial mon:stdio -pflash Outputs/qemu/SlimBootloader.bin
 
 Build Instruction for Slim Bootloader for LeafHill (APL) target
---------------------------------------------------------------
+---------------------------------------------------------------
 
 LeafHill is using PCI UART2 device as a serial port.
 For MEM32 serial port, CONFIG_SYS_NS16550_MEM32 needs to be enabled in U-Boot.
-- 
2.20.1

